lib/pages/settings/settings.dart

```diff
@@ -163,8 +168,15 @@ class SettingsController extends State<Settings> with ConnectPageMixin {
 
   @override
   void initState() {
-    WidgetsBinding.instance.addPostFrameCallback((_) => checkBootstrap());
-
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      final client = Matrix.of(context).client;
+      profileFuture ??= client.getProfileFromUserId(
```

Comment:
why need change that, Khai did the same thing, it can be conflict

Reply:
Before that, it's called in `build` func => Bad.

I move it to `initState` and `build` func just only focus gen UI

<img width="475" alt="Screenshot 2023-09-22 at 09 49 32" src="https://github.com/linagora/twake-on-matrix/assets/99852347/787213df-fc00-4ee9-9868-08e1f337d378">